appengine/standard/endpoints/backend/echo-v1_swagger.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
In the flexible endpoints sample, we use yaml for the definition. Is it possible to use yaml here as well or not?